homeassistant/components/freebox/.coveragerc

```diff
@@ -0,0 +1 @@
+camera.py
```

Comment:
You should not create a new file but use the existing one … at the root of the project ! 
Path: `/.coveragerc`

Linked to https://github.com/home-assistant/core/pull/88104/files#r1143114319

Reply:
Done